### DISCUSSION TITLE
Support mobile view

### DISCUSSION THREAD

**Discussion by kamenskiyyyy** [keystonejs/keystone#8070] [Feature requests]
Now the interface is uncomfortable to use on mobile devices.

You have to reduce the screen zoom, but then everything becomes shallow😢

<img src="https://user-images.githubusercontent.com/72261703/200122965-d22abd1e-6bf7-4a3c-949c-8218c8938f9f.PNG" width="200"/>

**Comment by flexdinesh** [comment]
There's another discussion for responsive Admin UI. 

- #7495
